src/frontend/components/DashboardVideoLiveControlPane/index.tsx

```diff
@@ -14,28 +18,53 @@ import { DashboardVideoLiveWidgetVisibilityAndInteraction } from './widgets/Dash
 import { DashboardVideoLiveWidgetVOD } from './widgets/DashboardVideoLiveWidgetVOD';
 import { DashboardVideoLiveWidgetSharedLiveMedia } from './widgets/DashboardVideoLiveWidgetSharedLiveMedia';
 
+const messages = defineMessages({
+  titleConfiguration: {
+    defaultMessage: 'CONFIGURATION',
```

Comment:
IMO, we should not use upper-cased messages.
I think we should keep it capitalized, and use grommet to upper-case it through css.